depends/patches/qt/fix_darwin_archs.patch

```diff
@@ -0,0 +1,16 @@
+To cross compile for the x86_64-darwin host on aarch64-darwin,
+passing QMAKE_APPLE_DEVICE_ARCHS=arm64 to the configure script
+does not work for some reasons.
+This is a simpliest workaround for this issue.
+
+--- old/qtbase/mkspecs/common/macx.conf
++++ new/qtbase/mkspecs/common/macx.conf
+@@ -6,7 +6,7 @@
+ QMAKE_MAC_SDK           = macosx
+ 
+ QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.12
+-QMAKE_APPLE_DEVICE_ARCHS = x86_64
++QMAKE_APPLE_DEVICE_ARCHS = $$system("uname -m")
```

Comment:
Right, we can't do that because we only use this for the Linux cross-compile (`macx-clang-linux`). Kind of annoying that this needs even more hacky patching. Although I guess that's what you get when trying to force support for things like this into Qt versions that well and truly don't support it.